crates/catalog/rest/src/catalog.rs

```diff
@@ -134,6 +134,22 @@ impl RestCatalogConfig {
             Client::builder().default_headers(headers).build()?,
         ))
     }
+
+    fn extract_optional_oauth_params(&self) -> HashMap<&str, &str> {
```

Comment:
minor: we could simplify the method name to `optional_oauth_params`, which is consistent with other names, like `table_endpoint`

Reply:
+1 for this naming.